common/src/main/java/com/redhat/cloud/notifications/routers/sources/SecretUtils.java

```diff
@@ -225,11 +225,29 @@ private long createSecretTokenSecret(final String secretToken) {
      * @param basicAuthentication the object to check.
      * @return {@code true} if the object is null, or if the password and the username are blank.
      */
-    private boolean isBasicAuthNullOrBlank(final BasicAuthentication basicAuthentication) {
+    protected boolean isBasicAuthNullOrBlank(final BasicAuthentication basicAuthentication) {
         if (basicAuthentication == null) {
             return true;
         }
 
-        return basicAuthentication.getUsername().isBlank() && basicAuthentication.getPassword().isBlank();
```

Comment:
I think we have access to apache's [StringUtils.isBlank](https://commons.apache.org/proper/commons-lang/javadocs/api-3.1/org/apache/commons/lang3/StringUtils.html#isBlank(java.lang.CharSequence))
We could use that one instead.

Reply:
Thanks, yeah! I like it more since it makes the check much more simple.